File: Proyectos_python/Conversor_calificaciones.py
print("Este es un programa de conversion de calificaciones")
print("Ingresa tu calificación del 0 al 100")
calificacion=int(input())
if 97 <= calificacion <=100:
    calificacion_alfabetica="A+"
elif 93 <= calificacion <=96:
    calificacion_alfabetica="A"
elif 90 <= calificacion <=92:
    calificacion_alfabetica="A-"
elif 87<= calificacion<= 89:
    calificacion_alfabetica="B+"
elif 83 <= calificacion <= 86:
    calificacion_alfabetica="B"
elif 80 <= calificacion <= 82:
    calificacion_alfabetica="B-"
elif 77 <= calificacion <= 79:
    calificacion_alfabetica="C+"    
elif 73 <= calificacion <= 76:
    calificacion_alfabetica="C"
elif 70 <= calificacion <= 72:
    calificacion_alfabetica="C-"
elif 67 <= calificacion <= 69:
    calificacion_alfabetica="D+"
elif 65 <= calificacion <=66:
    calificacion_alfabetica="D"
elif 0 <= calificacion <=64:
    calificacion_alfabetica="D-"
print(f"Tu calificacion en numeros es {calificacion} y tu calificacion alfabetica es {calificacion_alfabetica}")

File: Proyectos_c/conversor_calificaciones.c

#include <stdio.h>

int main()
{
int calificacion;
printf("Ingresa tu calificacion del 0 al 100\n");
scanf("%d", &calificacion);
if (calificacion>=97 && calificacion <=100){
    printf("Tu calificacion es A+");
    
}else if (calificacion>=93 && calificacion <=96){
    printf("Tu calificacion es A");
}else if (calificacion>=90 && calificacion <=92){
    printf("Tu calificacion es A-");
}else if (calificacion>=87 && calificacion <=89){
    printf("Tu calificacion es B+");
}else if (calificacion>=83 && calificacion <=86){
    printf("Tu calificacion es B");
}else if (calificacion>=80 && calificacion <=82){
    printf("Tu calificacion es B-");
}else if (calificacion>=77 && calificacion <=79){
    printf("Tu calificacion es C+");
}else if (calificacion>=73 && calificacion <=76){
    printf("Tu calificacion es C");
}else if (calificacion>=70 && calificacion <=72){
    printf("Tu calificacion es C-");
}else if (calificacion>=67 && calificacion <=69){
    printf("Tu calificacion es D+");
}else if (calificacion>=65 && calificacion <=66){
    printf("Tu calificacion es D");
}else if (calificacion>=0 && calificacion <=64){
    printf("Tu calificacion es D-");
}
}
